helloworld.py

```diff
@@ -1 +1,3 @@
-print("Hello World!")
+import sys
+
+print("Hello {}!".format(sys.argv[1]))
```

Comment:
I do not really appricate giving sys.argv[1] expression directly in print statement. It would better to use variable.